lib/pfconfig/objects/Net/Netmask.pm

```diff
@@ -0,0 +1,23 @@
+package pfconfig::objects::Net::Netmask;
+
+use base qw(Net::Netmask);
+
+sub new {
+    my $package = shift;
+    my $o = Net::Netmask->new(@_);
+    return bless $o, $package;
```

Comment:
do we really want to allow that ? 

don't think its in our usual practice to do it from the other constructors I looked at